src/Phansible/Roles/Xdebug.php

```diff
@@ -34,4 +36,17 @@ public function getInitialValues()
           'settings' => [],
         ];
     }
+
+    public function transformValues(array $values, VagrantBundle $vagrantBundle)
+    {
+        if ($values['install'] == 1) {
+            $values['settings'] = [
```

Comment:
Using defaults sounds like a good way to prevent both hidden fields and `RoleValuesTransformer ` - will have a look.